src/main.rs

```diff
@@ -48,7 +48,7 @@ enum SubCommand {
 
 impl SubCommand {
     fn from_string(name: &String, other_args: Vec<String>) -> Self {
-        return match name.to_owned().trim() {
```

Comment:
I'm a big fan of explicit returns, even in Rust where that really goes against the standard "style". Perhaps it's just because I'm too "Pythonic" now, but I think "explicit is better than implicit" is just a good way to write code.

Reply:
It looks like I messed up too and forgot the semi-colon... not sure why this compiles actually 🤷

Reply:
The ; is actually not needed I learned just now! The ; will make it return (), but because you early return by using the `return` keyword it never even reaches the ;. See also [this comment on reddit](https://www.reddit.com/r/rust/comments/v7x8my/comment/ibn8zsy/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button). Without the "early return" it would therefore fail if you include the ;. 

I put the returns back in for now, although I do like that its very easy to spot the early returns if you only use return for that :p.